crates/partition-store/src/durable_lsn_tracking.rs

```diff
@@ -136,7 +136,13 @@ impl EventListener for DurableLsnEventListener {
                 shared_state.note_durable_lsn(*partition_id, *lsn);
             } else {
                 warn!(
-                    cf_name = flush_job_info.cf_name,
+                    cf_name = std::str::from_utf8(
+                        flush_job_info
+                            .cf_name()
```

Comment:
Upstream rust-rocksdb has gained `FlushEvent` support since 0.42, however the API differs slightly in that it returns a `Vec<u8>` compared to our old implementation: https://github.com/zaidoon1/rust-rocksdb/blob/f61449b4fca345187a79d08070ed418f332f029f/src/event_listener.rs#L10